tables/output_tables.Rmd

```diff
@@ -143,19 +143,20 @@ soi_df <- tmb_hist %>%
     ),
     # add Lynch syndrome
     cancer_predispositions = case_when(
-      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "NF-1, Lynch syndrome",
+      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "Lynch syndrome",
```

Comment:
I'm assuming NF-1 was removed here because of something in a separate path report or similar?

Reply:
Nope, it's because of this! https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1591#issuecomment-1206493077

👍